db/migrate/20160925060545_change_column_name_in_pieces.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnNameInPieces < ActiveRecord::Migration
+  def change
+  	rename_column :pieces, :type, :piece_name
```

Comment:
You are correct that `type` is used for STI & that the validations are already under the hood ;)